docs/medical-api/more-info/ai-summary.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: "AI Summary"
+description: "How to use the AI Summary feature"
+---
```

Comment:
for everywhere:

we need to decide whether this is going to be an "AI Summary" or an "AI Brief", or something else - and be consistent

Reply:
Since we say MR Summary - I think AI Brief makes sense